test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -450,39 +450,38 @@ object TestAspect extends TimeoutVariants {
    * An aspect that restores the [[TestClock]]'s state to its starting state after the test is run.
    * Note that this is only useful when repeating tests.
    */
-  def restoreTestClock = restore[TestClock](_.clock)
+  def restoreTestClock: TestAspect[Nothing, TestClock, Nothing, Any, Nothing, Any] = restore[TestClock](_.clock)
```

Comment:
We may want to use the `TestAspectAtLeastR` alias here. I think that could help us clean up some of these other type signatures as well.

Reply:
I missed that comment about `TestAspectAtLeastR` before. I've done a couple of the other obvious ones while I was at it.